### PR TITLE
Fix flight

### PR DESCRIPTION
**Pull request by Zereff06**
1.Fixed if you are not wearing 2 air aspect armor, the armor disables all flying. 
 2. Fixed  if you remove the armor with air aspect while in flight, flying does not stop.

### CODE REVIEW

**Comment by Dream-Master** [comment]
not sure if you can fix with spotless that it not build

**Review by Shibva — APPROVED** [review]
appears to have been successfully built and tested

(note: if I shoudnt be allowed to have the ability to approve this then might be best to fix that)

**Comment by BlueWeabo** [comment]
> appears to have been successfully built and tested
> 
> (note: if I shoudnt be allowed to have the ability to approve this then might be best to fix that)

Even if you have the ability to approve. you need to have write access to the gtnh repo for it to matter.

File: src/main/java/witchinggadgets/common/items/armor/ItemPrimordialArmor.java [hunk 1]
```diff
@@ -186,8 +191,15 @@ public void onArmorTick(World world, EntityPlayer player, ItemStack stack) {
             player.addPotionEffect(new PotionEffect(Potion.fireResistance.id, 202, 0, true));
         }
 
-        if (amorcounter >= 2 && modescounter[0] >= 2) player.capabilities.allowFlying = true;
-        else player.capabilities.allowFlying = false;
+        // turn flight off or on
+        if ((amorcounter >= 2 && modescounter[0] >= 2)) {
+            flightStatus = FlightStatus.ON;
+            player.capabilities.allowFlying = true;
+        } else if (flightStatus.equals(FlightStatus.ON)) {
```
Comment (glowredman) [inline src/main/java/witchinggadgets/common/items/armor/ItemPrimordialArmor.java:198]: Use `==` instead of `equals()`, you're comparing an enum.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**